sds_data_manager/constructs/ialirt_api_manager_construct.py

```diff
@@ -0,0 +1,88 @@
+"""Configure the I-ALiRT API Manager."""
+
+import aws_cdk as cdk
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_lambda as lambda_
+from constructs import Construct
+
+from .api_gateway_construct import ApiGateway
+
+
+class IalirtApiManager(Construct):
+    """Construct for API Management."""
+
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        code: lambda_.Code,
+        api: ApiGateway,
+        env: cdk.Environment,
+        data_bucket,
+        vpc,
+        layers: list,
+        **kwargs,
+    ) -> None:
+        """Initialize the SdsApiManagerConstruct.
+
+        Parameters
+        ----------
+        scope : obj
+            Parent construct
+        construct_id : str
+            A unique string identifier for this construct
+        code : lambda_.Code
+            Lambda code bundle
+        api : obj
+            The APIGateway stack
+        env : obj
+            The CDK environment
+        data_bucket : obj
+            The data bucket
+        vpc : obj
+            The VPC
+        layers : list
+            List of Lambda layers arns
+        kwargs : dict
+            Keyword arguments
+        """
+        super().__init__(scope, construct_id, **kwargs)
+
+        s3_read_policy = iam.PolicyStatement(
+            effect=iam.Effect.ALLOW,
+            actions=["s3:ListBucket"],
+            resources=[
+                data_bucket.bucket_arn,
+            ],
+        )
+
+        s3_read_policy.add_actions("s3:GetObject")
+        s3_read_policy.add_resources(f"{data_bucket.bucket_arn}/*")
```

Comment:
Is there a reason you've broken these additions to the read policy into these two additional lines, rather than including them in the initialization of the policy statement above?